### PR TITLE
AI Tutor Layout Anpassung

### PR DESCRIPTION
**Pull request by JonathanWolp**
# Description of changes

Please provide a short description of the change or if the change is sufficiently described in the GitHub issue, provide a link to the issue.

Anpassungen des Tutor-Chat Widget auf basis der RM des Spring Reviews:

Layout anpassungen beinhaltet:
nicht mehr transparent, Symbol verschiebt sich nicht mehr, wenn Chat länger wird
- https://gamify-it.gropius.dev/projects/ebc15ee5-70fe-49b6-9c3f-6e7dd23a1304/issues/f8bf4229-9e7e-4d61-91e6-3acea694f019

Tutor ist in der lecturer view nicht mehr sichtbar:
- https://gamify-it.gropius.dev/projects/ebc15ee5-70fe-49b6-9c3f-6e7dd23a1304/issues/6707d67d-43ee-4211-9eaf-5f7f22af50fc

## How has this been tested?

Please describe the test strategy you followed

- [ ] automated unit test
- [ ] automated integration test
- [ ] automated acceptance test
- [ x] manual, exploratory test

In case of manual test, please document the test well including a set of user instructions and prerequisites. Each including an action, it's result, and where appropriate a screenshot.

## Checklist before requesting a review

- [x] My code follows the [coding guidelines](https://github.com/MEITREX/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code fulfilles all acceptance criteria
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have made corresponding changes to the documentation
- [x] I have added explanation of architectural decision and rationales to [wiki/adr](https://github.com/MEITREX/wiki/tree/main/adr)
- [x] I have updated the changes in the ticket description

## Checklist for reviewer

- [x] The code works and does not throw errors
- [x] The code is easy to understand and there are no confusing parts
- [x] The code follows the [coding guidelines](https://github.com/MEITREX/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [x] The code change accomplishes what it is supposed to do
- [x] I cannot think of any use case in which the code does not behave as intended
- [x] The added and existing tests reasonably cover the code change
- [x] I cannot think of any test cases, input or edge cases that should be tested in addition
- [x] Description of the change is included in the documentation

### CODE REVIEW

File: components/tutor/TutorWidget.tsx [hunk 1]
```diff
@@ -136,12 +135,13 @@ export default function TutorWidget({ isAuthenticated }: TutorWidgetProps) {
   let style: React.CSSProperties = {
     position: "fixed",
     zIndex: 10000,
+    backgroundColor: "#fff",
```
Comment (ZiyaSelim) [inline components/tutor/TutorWidget.tsx:138]: Durch diesen background hat das ganze Widget immer einen Kasten um sich

**Review by ZiyaSelim — CHANGES_REQUESTED** [review]
Bis auf den einen Background der zu einem weißen Kasten führt sieht es gut aus!

File: components/tutor/TutorWidget.tsx (hunk 1 above)
Comment (JonathanWolp) [reply components/tutor/TutorWidget.tsx]: Zeile wurde gelöscht, um Background um das Widget zu vermeiden
